common/djangoapps/student/models.py

```diff
@@ -1381,6 +1384,22 @@ def refundable(self):
         else:
             return True
 
+    def _refund_window_end_date(self):
+        """ Calculate and return the refund window end date. """
+        order_number = CourseEnrollmentAttribute.get_enrollment_attribute(
+            self,
+            namespace='order',
+            name='order_number'
+        ).get('value')
+
+        order = ecommerce_api_client(self.user).orders(order_number).get()
```

Comment:
What happens if there's no corresponding  `CourseEnrollmentAttribute` and `order_number` is `None`? Will an exception be thrown?